fixtures/convert_to_fixtures.py

```diff
@@ -0,0 +1,20 @@
+
+'''convert surah.json to fixtures from
```

Comment:
I don't understand what are your fixtures. In Python in general scripts have the following outline:
```python
# Imports
import json

import click

# Functions implementing the main logic
def convert():
    pass

# Main function which parses command line arguments
@click.command()
@click.option('--count', default=1, help='number of greetings')
@click.argument('name')
def main(count, name):
    pass

if __name__ == "__main__":
    main()
```

Reply:
Used this to convert external surah.json file from quranapiproject to fixtures files, which can populate our database. Kept this for futures additions, ec we decide to add arabic name of surah, juz, other data.

Reply:
I’m not saying ”delete this script”, just write it in a proper way :)